manufacturing/filter_holder/README.md

```diff
@@ -65,6 +65,14 @@ tubing, and the other - with vents for air in/out-flow.
 
 Most recent changes also include thickening of the walls to make the 3d prints less fragile.
 
+### Lower Face Plate
+
+This is a clear acrylic sheet that houses the filter assembly. It is mounted using heat inserts. Link to the part file is given in [Source files](#Source-files) below. Some other useful files for the same part are [.idw 2D drawing](CAD/LowerAcrylicFacePlate.idw), [.pdf 2D drawing](CAD/LowerAcrylicFacePlate.pdf),[.stl for 3D printing](CAD/LowerAcrylicFacePlate.stl),[.dxf of the front face for laser cutting](CAD/LowerAcrylicFacePlate_FrontFace.dxf).
+
+![](CAD/LowerAcrylicFacePlate.png)
```

Comment:
I would use jpg instead of png for rendering screenshots. Better compression and quality.

You could also rotate the rendering so it is more horizontal. That is a lot of screen real estate for a diagonal thing.

Reply:
> I would use jpg instead of png for rendering screenshots. Better compression and quality.
> 
> You could also rotate the rendering so it is more horizontal. That is a lot of screen real estate for a diagonal thing.

Done